### ISSUE TITLE
catkin_prepare_release raised an exception

### ISSUE THREAD

**Issue by iquater** [ros-infrastructure/catkin_pkg#341]
Hi!
I met an issue in a last catkin-pkg release (0.5.1) (Ubuntu focal 20.04)
When I try to execute 
`catkin_prepare_release --no-push --no-color --non-interactive`
an exception is raised:
```
  File "/usr/bin/catkin_prepare_release", line 33, in <module>
    sys.exit(load_entry_point('catkin-pkg==0.5.1', 'console_scripts', 'catkin_prepare_release')())
  File "/usr/lib/python3/dist-packages/catkin_pkg/cli/prepare_release.py", line 217, in main
    _main()
  File "/usr/lib/python3/dist-packages/catkin_pkg/cli/prepare_release.py", line 272, in _main
    build_types = package.get_build_types()
  File "/usr/lib/python3/dist-packages/catkin_pkg/package.py", line 124, in __getattr__
    raise AttributeError(name)

```
There is no `get_build_types` method in the `catkin_pkg.package` module, seems `get_unconditional_build_types` should be used.

https://github.com/ros-infrastructure/catkin_pkg/pull/337/files

**Comment by cottsay** [comment]
You're right - this was caused by #340, which should have been updated after renaming the function in the last commit to #337.

**Comment by nuclearsandwich** [comment]
:facepalm:

**Comment by nuclearsandwich** [comment]
Thanks for the report. Boy do I feel like a dope. https://github.com/ros-infrastructure/catkin_pkg/pull/342 is opened and I'll take care of testing it from source before tagging 0.5.2.

**Comment by nuclearsandwich** [comment]
The fix has been released in catkin_pkg 0.5.2.
